src/ImageLoaders/IHex32/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Reflection;
```

Comment:
I would have added some comments here, but since I'm advocating moving all of this code into the existing `IntelHexLoader` assembly those comments would be redundant.